src/api/.keystone/config.js

```diff
@@ -102,14 +102,36 @@ var notify = {
         },
         reference: null
       });
-      return { success: true };
+      return {
+        success: true,
+        emailRecipient: sendToEmailAddress
+      };
     } catch (err) {
       throw new Error(`Calling Notify API. Unable to send email ${err}`);
     }
   }
 };
 var notify_default = notify;
 
+// emails/index.ts
+var confirmEmailAddress = async (email, firstName, verificationHash) => {
+  try {
+    console.info("Sending email verification for account creation");
+    const emailResponse = await notify_default.sendEmail(EMAIL_TEMPLATE_IDS.ACCOUNT.CONFIRM_EMAIL, email, firstName, verificationHash);
+    if (emailResponse.success) {
+      return emailResponse;
+    }
+    throw new Error(`Sending email verification for account creation ${emailResponse}`);
+  } catch (err) {
+    console.error(err);
+    throw new Error(`Sending email verification for account creation ${err}`);
+  }
+};
+var sendEmail = {
```

Comment:
should this be `let`/`const`?

Reply:
This is auto generated by keystone from our typescript file

Reply:
Resolving this so we can merge - I did try adding this to `.gitignore`, but turns out this is required for the build unfortunately.